modules/installation-create-ingress-dns-records.adoc

```diff
@@ -0,0 +1,114 @@
+router-default   LoadBalancer   172.30.62.215   ab3...28.us-east-2.elb.amazonaws.com   80:31499/TCP,443:30693/TCP   5m
+----
+
+. Locate the hosted zone ID for the load balancer:
++
+----
+$ aws elb describe-load-balancers | jq -r '.LoadBalancerDescriptions[] | select(.DNSName == "<external_ip>").CanonicalHostedZoneNameID' <1>
+
+Z3AADJGX6KTTL2
+----
+<1> For `<external_ip>`, specify the value of the external IP address of the Ingress Operator load balancer that you obtained.
++
+The output of this command is the load balancer hosted zone ID.
+
+. Obtain the public hosted zone ID for your cluster's domain:
++
+----
+$ aws route53 list-hosted-zones-by-name \
+            --dns-name "<domain_name>" \ <1>
+            --query 'HostedZones[? Config.PrivateZone != `true` && Name == `<domain_name>.`].Id' <1>
+            --output text
+
+/hostedzone/Z3URY6TWQ91KVV
+----
+<1> For `<domain_name>`, specify the Route53 base domain for your {product-title} cluster.
++
+The hosted zone for your domain is shown in the command output. In this example, it is `Z3URY6TWQ91KVV`.
+
+. Add the alias records to your private zone:
++
+----
+$ aws route53 change-resource-record-sets --hosted-zone-id "<private_hosted_zone_id>" --change-batch '{ <1>
+>   "Changes": [
+>     {
+>       "Action": "CREATE",
+>       "ResourceRecordSet": {
+>         "Name": "\\052.apps.<cluster_domain>", <2>
+>         "Type": "A",
+>         "AliasTarget":{
+>           "HostedZoneId": "<hosted_zone_id>", <3>
+>           "DNSName": "<external_ip>.", <4>
+>           "EvaluateTargetHealth": false
+>         }
+>       }
+>     }
+>   ]
+> }'
+----
+<1> For `<private_hosted_zone_id>`, specify the value from the output of the CloudFormation template for DNS and load balancing.
+<2> For `<cluster_domain>`, specify the domain or subdomain that you use with your {product-title} cluster.
+<3> For `<hosted_zone_id>`, specify the hosted zone ID for your cluster.
+<4> For `<external_ip>`, specify the value of the external IP address of the Ingress Operator load balancer. Ensure that you include the trailing period (`.`) in this parameter value.
+
+. Add the records to your public zone:
++
+----
+$ aws route53 change-resource-record-sets --hosted-zone-id "<public_hosted_zone_id>"" --change-batch '{ <1>
+>   "Changes": [
+>     {
+>       "Action": "CREATE",
+>       "ResourceRecordSet": {
+>         "Name": "\\052.apps.<cluster_domain>", <2>
+>         "Type": "A",
+>         "AliasTarget":{
+>           "HostedZoneId": "<hosted_zone_id>", <3>
+>           "DNSName": "<external_ip>.", <4>
+>           "EvaluateTargetHealth": false
+>         }
+>       }
+>     }
+>   ]
+> }'
+----
+<1> For `<public_hosted_zone_id>`, specify a the public hosted zone for your domain.
+<2> For `<cluster_domain>`, specify the domain or subdomain that you use with your {product-title} cluster.
+<3> For `<hosted_zone_id>`, specify the hosted zone ID for your cluster.
```

Comment:
So is this one, does hosted_zone_id mean hosted zone ID for the load balancer in step 3?